django_project/core/base_static/css/sidebar.css

```diff
@@ -31,15 +32,17 @@
 }
 
 #sidebar ul li {
-    padding-left: 1rem;
-    padding-top: 0.5rem;
     width: 100%;
     list-style: none;
     cursor: pointer;
     background-color: rgba(255, 255, 255, 0.13);
-    height: 2.5rem;
+    height: 100%;
     line-height: 1.25rem;
     color: var(--c-grey-l);
+    padding-left: 1rem;
```

Comment:
you can replace these four lines to one:
`padding: 0.5rem 1rem`

padding: top right bottom left